docs/pages/guides/local-app-development.mdx

```diff
@@ -71,6 +71,24 @@ If you install [`expo-dev-client`](/develop/development-builds/introduction/#wha
 
 To create a development build, you can use [local app compilation](#local-app-compilation) commands (`npx expo run:[android|ios]`) which will create a debug build and start the development server.
 
+## Local builds using Android product flavors
+
+> **warning** This feature is only available for SDK 52 and above.
+
+If you have a custom Android project with multiple product flavors using different application ids, you can configure `npx expo run:android` to use the correct flavor and build type. Expo supports both `--variant` and `--app-id` to customize the build and launch behavior when building for Android.
+
+The `--variant` flag can be used to switch the Android build type from **debug** to **release**. This flag can also be used to configure a product flavor and build typ, formatted in camelCase. For example, if you have both a **free** and **paid** product flavor, as documented on [Android developers](https://developer.android.com/build/build-variants#change-app-id), you can build a development version of your app with:
```

Comment:
```suggestion
The `--variant` flag can switch the Android build type from **debug** to **release**. This flag can also configure a product flavor and build type, when formatted in camelCase. For example, if you have both [**free** and **paid** product flavors](https://developer.android.com/build/build-variants#change-app-id), you can build a development version of your app with:
```

- Remove passive voice "can be used ...". The sentences read fine without it.
- Fix typo in the second sentence.
- In third sentence, let's add the link to product flavors text instead of explicitly mentioning Android docs. It helps us avoid breaking the flow of text, and if a reader is curious about where they should learn about product flavors, they can click that hyperlink.